### DISCUSSION TITLE
How do I flash a real N64 Cartridge

### DISCUSSION THREAD

**Discussion by jeremy64** [sanni/cartreader#406] [Need help]
I was looking all over the internet for this but I can't seem to find the answer to this. Is it possible to use the cart reader to write a ROM/ROM hack to a real N64 cart?

**Comment by sanni** [comment]
You can't flash a real N64 cartridge but you can flash those fake ones that are all over eBay. 

Well theoretically, I have toyed around with that in 2017 but I have not had a look since then and it could very well be that the Chinese manufacturers have changed their design and/or the flashroms they use, so you would need to adapt the flash functions in the Cart Readers source code.

Honestly it's not worth the trouble when you can get an Everdrive 64/Super 64/ED64+ for cheap these days. Especially since the Chinese repros often use salvaged flash chips that could break any time.

**Reply by jeremy64** [reply]
Oh ok. Thank you!
